examples/integration/xcodeproj_targets.bzl

```diff
@@ -18,8 +18,9 @@ XCODEPROJ_TARGETS = [
     ),
     "//iMessageApp",
     top_level_target(
-        label = "//iOSApp",
+        label = "//iOSApp/Source:iOSApp",
```

Comment:
This was needed because `//iOSApp` is an `alias` and I'm not sure we can get the comparison between labels to work correctly if they're an `alias` (similar to how we enforce non-`alias` labels in the schemes API). Thoughts?

Reply:
Correct, we can't resolve aliases in Starlark. Since we already have that restriction, this seems fine. Let's make sure we have the same `fail()` error when this is an alias.